NDTensors/src/lib/LabelledNumbers/src/labelledinteger.jl

```diff
@@ -3,13 +3,19 @@ struct LabelledInteger{Value<:Integer,Label} <: Integer
   label::Label
 end
 LabelledStyle(::Type{<:LabelledInteger}) = IsLabelled()
+# TODO: Define `set_value` and `set_label`?
 label(lobject::LabelledInteger) = lobject.label
 # TODO: Use `TypeParameterAccessors`.
```

Comment:
Module `Sector` also defined a `label` function. I think they be unified as 2 methods of the same function.

Reply:
Agreed, let's address that in a future PR.